### PR TITLE
Feature/loader animations

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Added a loading screen animation on page reload
- used a setTimeout of 1400ms to allow for time to view animation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" type="images/png" href="./Images/favicon.png" />
     <link rel="manifest" href="./manifest.json" />
-    <title>React App</title>
+    <title>PokeDex</title>
```
Comment (hayleyw7) [inline public/index.html:11]: Thx for catching that.

File: src/components/App/App.js [hunk 2]
```diff
@@ -75,8 +76,11 @@ const App = () => {
       <Header />
       <Route exact path='/'
         render={() =>
+          
           <main className='main-content'>
-            <Search addPokemon={addPokemon} clearErrorMessage={clearErrorMessage}/>
+            {!pokeDex.length && <Loader />}
+            {pokeDex.length && <Search addPokemon={addPokemon} clearErrorMessage={clearErrorMessage}/>}
```
Comment (hayleyw7) [inline src/components/App/App.js:82]: Glad we got it to hide during Loader.

File: src/components/Loader/Loader.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import './Loader.css'
+import { gsap } from 'gsap'
+
+const Loader = () => {
+  let loadingMessage = useRef(null)
+  let pokeball = useRef(null)
+
+  useEffect(() => {
+
+    gsap.from([pokeball, loadingMessage], {
```
Comment (hayleyw7) [inline src/components/Loader/Loader.js:11]: This is SO GREAT!

**Review by hayleyw7 — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (joanafbrito) [reply public/index.html:11]: 🤗🤗

**Review by joanafbrito — APPROVED** [review]
Love the Loading , thanks for doing that

**Pull request merged**